server/src/test/java/io/crate/execution/engine/aggregation/impl/AverageAggregationTest.java

```diff
@@ -169,7 +169,7 @@ public void testInterval() throws Exception {
             {Period.days(4).withHours(7).withMinutes(39).withSeconds(45)},
             {Period.days(2).withHours(27).withMinutes(52).withSeconds(25)}
         }))
-            .isEqualTo(new Period(0, 0, 0, 763, 14, 54, 26, 0,
+            .isEqualTo(new Period(0, 0, 0, 773, 14, 54, 26, 0,
```

Comment:
Is it days unit? Looks like quite discrepancy with the prev value?

Reply:
Yes, and it was before because of: https://github.com/crate/crate/pull/14124/commits/47030fbfa59a1e58656d54df2dd13d0c80324d94#diff-2bc8339df85d5229dc575d383b2f6b8aaeb3925e83d678b2822cd21a5a59d173R176
And now the standard duration is not used at all (before used for serializing to a BigInteger).

psql returns:
```
select avg(x) from unnest(array[
                INTERVAL '6 years 3 mons',
                INTERVAL '4 years 3 hours',
                INTERVAL '4 mons 12 hours',
                INTERVAL '4 days 7 hours 39 mins 45 secs',
                INTERVAL '2 days 27 hours 52 mins 25 secs'
            ]) t (x) order by 1 desc;
              avg               
--------------------------------
773 days 14:54:26
(1 row)
```